admin_honeypot/views.py

```diff
@@ -36,6 +38,9 @@ def get_context_data(self, **kwargs):
             'app_path': path,
             REDIRECT_FIELD_NAME: reverse('admin_honeypot:index'),
             'title': _('Log in'),
+            'site_title': admin.site.site_title,
```

Comment:
As per dmpayton's rough confirmation on #62, I _think_ this should be `AdminSite().site_title` and `AdminSite().site_header` to avoid leaking any custom text a user has assigned to the 'default' adminsite instance; eg, compare:
```
>>> from django.contrib import admin
>>> admin.site.site_title = 'test'
>>> str(admin.AdminSite().site_title)
'Django site admin'
```

Reply:
@GitRon Please see kezabelle's review.

Reply:
@Keda87 @9mido Done 🙃

Reply:
@GitRon Thanks hero :)